public_html/wp-content/plugins/wordcamp-coming-soon-page/classes/wordcamp-coming-soon-page.php

```diff
@@ -263,7 +263,7 @@ public function get_dates() {
 	}
 
 	/**
-	 * Retrieve the contact form shortcode string
+	 * Loop through all pages and renders first contact-us form or contact-us block
```

Comment:
I think the short description is meant to say _what_ a function does, rather than _how_ it does it, so I'd leave out the bit about looping through the pages, since that's just an implementation detail. Maybe something more along the lines of "Retrieve the contact form shortcode/block output" ?

Reply:
I guess in this case I would keep the Loop through pages bit as well, because this methodology is a but unusual and unexpected. Just saying `Retrieve the contact form shortcode/block output` implies that there is a dedicated shortcode or block somewhere just for this contact form. Explaining that we loop through pages could clearly help someone in debugging issues related to this function.